types/index.d.ts

```diff
@@ -1805,18 +1805,18 @@ declare module 'mongoose' {
 
     /** Specifies a `$gt` query condition. When called with one argument, the most recent path passed to `where()` is used. */
     gt(val: number): this;
-    gt(path: string, val: number): this;
+    gte<KEY extends keyof FilterQuery<DocType>>(path: KEY, val: FilterQuery<DocType>[KEY]['$gt']): this;
```

Comment:
Hey, I was just reading through the diffs and noticed this error.

You changed the name of the function. From `gt` to `gte` and there is now only one overload for the `gt` function.

Reply:
Ack! You're right, good catch! @vkarpov15 very sorry about that! I'm assuming you'll want to just fix it since it's merged already? Let me know if you need me for anything

Reply:
covered by #11760